articles/getting-started/getting-started-sparkpost.md

```diff
@@ -19,7 +19,7 @@ Note: SparkPost is a developer-centric email delivery service. That means pretty
 
 ## Signing Up And In
 
-First things first: sign up for your free SparkPost account [here](https://app.sparkpost.com/sign-up). After sign-up, SparkPost will send you a verification email; please do verify your email address so SparkPost can contact you if you need support later on and send you important service announcements.
+First things first: sign up for your free SparkPost account [here](https://app.sparkpost.com/join). After sign-up, SparkPost will send you a verification email; please do verify your email address so SparkPost can contact you if you need support later on and send you important service announcements.
```

Comment:
should we add the marketing query param here too?

Reply:
afaik we don't have a salesforce campaign for this and it will get attributed to the website by default. I'll talk with the team about if we want a campaign.

In the meantime, I think we should push this change out 🤔 yes?

Reply:
yeah, we can follow up with marketing about this. the link below has a query param, but the other links don't..